stories/InfoBlock.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { foundationPartial } from 'styles/foundation';
+import ebiGlobalStyles from 'ebi-framework/css/ebi-global.css';
+import fonts from 'EBI-Icon-fonts/fonts.css';
+
+const f = foundationPartial(ebiGlobalStyles, fonts);
+
+export default {
+  title: 'Basic UI/InfoBlock',
+};
+
+export const Basic = () => (
+  <div
+    className={f('callout', 'info')}
```

Comment:
It would be cool to have the different styles of callouts: https://get.foundation/sites/docs/callout.html#

Maybe use a select knob (https://github.com/storybookjs/storybook/tree/master/addons/knobs) as we are already using knobs (e.g MemberDBSelector)

Reply:
It's a cool idea. All the styles can be exposed including the close button.